en/simulation/README.md

```diff
@@ -140,32 +140,32 @@ The simulated camera is a gazebo plugin that implements the [MAVLink Camera Prot
 
 The same approach can be used by other simulators to implement camera support.
 
-## Running simulation on the remote server 
+## Running Simulation On The Remote Server 
```

Comment:
Just FYI, in this kind of capitalisation it is usual to not capitalise small connecting words - e.g. "on the"